modules/nf-core/mash/sketch/main.nf

```diff
@@ -7,7 +7,7 @@ process MASH_SKETCH {
         'biocontainers/mash:2.3--he348c14_1' }"
 
     input:
-    tuple val(meta), path(reads)
+    tuple val(meta), path(fasta)
```

Comment:
I'm unsure what this variable name should be since FASTA and FASTQ files can both be used as input.